howto/images/use-specific-release.md

```diff
@@ -0,0 +1,16 @@
+(howto-use-specific-release)=
+# How to use a specific release
+
+With every new Anbox Cloud release, updated images are published. By default, the latest image release is pulled by AMS, but you can request a specific release with the following syntax:
+
+    amc image add <local image name> <remote image name>@<release>
+
+For instance, to fetch the arm64 Android 13 image of the 1.24.2 release:
+
+    amc image add foobar jammy:android13:arm64@1.24.2-<hash>
```

Comment:
I'm not sure how users are supposed to retrieve the hash, though.

Reply:
They are documented at https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/component-versions/#anbox-images